source/libs/page-detect.ts

```diff
@@ -373,6 +373,11 @@ export const _isBranches = [
 export const isUserProfile = (): boolean => select.exists('.user-profile-nav');
 export const _isUserProfile = domBased;
 
+export const isUserProfileRepoTab = (): boolean => location.search.startsWith('?tab=repositories');
```

Comment:
This isn't enough. It even matches `https://github.com/?tab=repositories`

Reply:
When I wrote originally I added isuserprofile but I did not know how to test it